conf/base.config

```diff
@@ -23,12 +23,26 @@ process {
     errorStrategy = { task.exitStatus in [143,137] ? 'retry' : 'ignore' }
   }
 
+  //"make_classifier" is the older version (just one step) of preparing the database for classification
   withName: make_classifier {
     cpus = { check_max (1 * task.attempt, 'cpus' ) }
     memory = { check_max (42.GB * task.attempt, 'memory' ) }
     time = { check_max (12.h * task.attempt, 'time' ) }
   }
 
+  //"clfr_extract_seq" and "clfr_train" is the newer version, where database preparation is split in two
+  withName: clfr_extract_seq {
+    cpus = { check_max (1 * task.attempt, 'cpus' ) }
+    memory = { check_max (42.GB * task.attempt, 'memory' ) }
+    time = { check_max (12.h * task.attempt, 'time' ) }
+  }
+
+  withName: clfr_train {
+    cpus = { check_max (1 * task.attempt, 'cpus' ) }
+    memory = { check_max (42.GB * task.attempt, 'memory' ) }
+    time = { check_max (12.h * task.attempt, 'time' ) }
+  }
```

Comment:
Do these have identical resources allocated? Since you argue that you split the process into two, why not immediately adjusting resources?

Reply:
I have changed the resource allocation for the ampliseq specific uppmax-config, since that's where I've run this and looked at resource use (and there the first step takes almost 5 days but no memory, and the second step just some hours with more memory). I tried to figure out what to change here, but since the entire process seem to run very fast for you I did not know if the behaviour was the same there. Do you have any idea if the behaviour is similar when you run this? Extracting take a long time but little memory and training is fast with more memory?

Reply:
I did not investigate the resources that each step needs in its current form. But before I had this pipeline (and QIIME2 just started), I investigated with bash scripts for each step and sequence extraction needed like no time at all (few minutes) and training ~2.5h). On the two clusters that I used with this pipeline, the combined total time was always ~3h with peak memory usage at ~40GB. So the settings above were totally fine.

So for now these settings are ok, but please remember to review these in case you find a solution for uppmax, that are somewhat reasonable also for other users.